### ISSUE TITLE
This Script Dont Wors Now

### ISSUE THREAD

**Issue by zaintech01** [JohnHammond/msdt-follina#17]
![Screenshot from 2022-06-05 18-36-12](https://user-images.githubusercontent.com/92736474/172053617-52d67e31-b262-4344-95bc-0689f375dcac.png)

**Comment by jppdpf** [comment]
Well I'm having trouble in the exploit execution when opening the generated doc (i suspect that the Windows Defender got an update). This is what brought me to your issue. But your issue is entirely different. 

You error is about python reading you network interface **eth0**. 
Is says it expects 4 octets (which is ipv4 XXX.XXX.XXX.XXX).
My two cents:
1. Either you don't have an IPv4 associated on the interface, in which you would have an IPv6. 
2. Or your don't have a defined interface eth0.

Most likely it's the first one.

**Comment by Sussy-OS** [comment]
I have the same error

**Comment by Sussy-OS** [comment]
![image](https://user-images.githubusercontent.com/98842253/172066918-659532ff-a260-425b-b93a-be268270a3aa.png)

**Comment by ghost** [comment]
This is because of ipv6.

**Comment by NathanNeves** [comment]
@jppdpf  I'm also having the same problem  right now, currently my payload isn't working the document opens but the  payload doesn't run

**Comment by jppdpf** [comment]
@NathanNeves Yep, it's probably not a zero-day no more x) microsoft must have fixed it. Which is rather sad because I wanted this exploit for a project.
